tests/codeception/common/_support/FixtureHelper.php

```diff
@@ -58,6 +58,30 @@ public function globalFixtures()
     }
 
     /**
+     * Workaround used to load custom fixtures for the test
+     * @link https://github.com/Codeception/Codeception/issues/2258
+     * @link https://github.com/yiisoft/yii2/issues/5575
+     * @param array $fixtureConfig
+     */
+    public function loadFixturesForTest(array $fixtureConfig)
+    {
+        $fixtures = ArrayHelper::merge(
```

Comment:
Ok, you are right.
I haven't realised that I should make a 'FixtureTrait::createFixtures()' also public. After that everything works as expected, but the test becomes ugly:

    public function updateFixtures(FunctionalTester $I)
    {
        $fixtures = ArrayHelper::merge(
            $I->globalFixtures(),
            $I->fixtures(),
            ['user' => [
               'class' => UserFixture::className(),
               'dataFile' => '@tests/codeception/frontend/functional/fixtures/signup_cest.php',
           ]]
        );
        $I->loadFixtures($I->createFixtures($fixtures));
    }